src/server/examples/examples_test.go

```diff
@@ -73,3 +74,156 @@ func TestExampleTensorFlow(t *testing.T) {
 	require.NoError(t, c.DeleteRepo("GoT_train", false))
 	require.NoError(t, c.DeleteRepo("GoT_scripts", false))
 }
+
+func TestFruitStand(t *testing.T) {
+
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	c := getPachClient(t)
+	t.Parallel()
+
+	require.NoError(t, c.CreateRepo("data"))
+	repoInfos, err := c.ListRepo(nil)
+	require.NoError(t, err)
+	var repoNames []interface{}
+	for _, repoInfo := range repoInfos {
+		repoNames = append(repoNames, repoInfo.Repo.Name)
+	}
+	require.OneOfEquals(t, "data", repoNames)
+
+	cmd := exec.Command(
+		"pachctl",
+		"put-file",
+		"data",
+		"master",
+		"sales",
+		"-c",
+		"-f",
+		"https://raw.githubusercontent.com/pachyderm/pachyderm/master/doc/examples/fruit_stand/set1.txt",
```

Comment:
Won't this now fail, if you try to test a commit locally that hasn't been pushed to github?
I feel like doing the simple thing and just reading the copy of this file that's in the local filesystem as part of the commit would be a lot simpler and less fragile. The only thing it wouldn't cover is if github changes the format of their URLs, but I think that's acceptable.